common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -605,62 +651,54 @@ div.video {
           }
         }
 
-        div.lang {
-          & > a.hide-subtitles {
-            @extend %video-button;
-            @include transition(none);
-            box-shadow: inset 1px 0 0 #555;
-            background: url('#{$static-path}/images/cc.png') center no-repeat;
-            border-left: none;
-            border-right: none;
-            padding: 0 11px;
-            width: 30px;
-
-            &.off {
-              opacity: 0.7;
+        .toggle-transcript {
+
+            &.is-active {
+              color: rgb(14, 166, 236); // UXPL primary accent
             }
-          }
+        }
 
-          .menu.langs-list {
-            right: -1px;
-            width: 150px;
+        .lang {
+
+          & > .hide-subtitles {
+            @include transition(none);
           }
         }
       }
     }
 
-    &:hover section.video-controls {
-      ul, div {
-        opacity: 1;
-      }
+    &:hover {
 
-      div.slider {
-        @include transform(scaleY(1) translate3d(0, 0, 0));
+      .video-controls {
 
-        a.ui-slider-handle {
-          @include transform(scale(1) translate3d(-50%, -15%, 0));
+        .slider {
+          @include transform(scaleY(1) translate3d(0, 0, 0));
+
+          .ui-slider-handle {
+            @include transform(scale(1) translate3d(-50%, -25%, 0));
+          }
         }
       }
     }
   }
 
-  ol.subtitles {
-    padding-left: 0;
-    float: left;
-    max-height: 460px;
+  .subtitles {
+    @include float(left);
     overflow: auto;
-    width: flex-grid(3, 9);
     margin: 0;
+    max-height: 460px;
+    width: flex-grid(3, 9);
+    @include padding-left(0);
```

Comment:
This should actually be padding: 0; since it needs to override the bad ul, ol hardcoded padding-left and what we want here is no padding so it fits in the space. (Right now the RTL view had the transcript below the video.)